File: test/fixtures/node/get_with_form.js
import fetch, { FormData } from 'node-fetch';

const form = new FormData();
form.append('from', 'test@tester.com');
form.append('to', 'devs@tester.net');
form.append('subject', 'Hello');
form.append('text', 'Testing the converter!');

fetch('http://localhost:28139/v3', {
  method: 'POST',
  headers: {
    'Authorization': 'Basic ' + btoa('test:')
  },
  body: form
});

File: test/fixtures/julia/get_with_form.jl
using Base64, HTTP

headers = Dict(
    "Authorization" => "Basic " * base64encode("test:")
)

form = HTTP.Form(
    Dict(
        "from" => "test@tester.com",
        "to" => "devs@tester.net",
        "subject" => "Hello",
        "text" => "Testing the converter!"
    )
)

resp = HTTP.post("http://localhost:28139/v3", headers, form)
